index.bs

```diff
@@ -77,16 +77,18 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class.
+The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class or the {{RawMagnetometer}} class,
+depending on the need for calibrated readings or raw uncalibrated readings.
 
-The Magnetometer has a <a>default sensor</a>, which is the device's main magnetomer sensor.
+The {{Magnetometer}} and {{RawMagnetometer}} have an associated {{PermissionName}} which is <a for="PermissionName" enum-value>"magnetometer"</a>.
 
-The Magnetometer has an associated {{PermissionName}} which is <a for="PermissionName" enum-value>"magnetometer"</a>.
-
-A [=latest reading=] per [=sensor=] of magnetometer type includes three [=map/entries=]
+For {{Magnetometer}} class, the [=latest reading=] includes three [=map/entries=]
 whose [=map/keys=] are "x", "y", "z" and whose [=map/values=] contain current <a>magnetic field</a>
 about the corresponding axes.
 
+For {{RawMagnetometer}} class, the [=latest reading=] includes three [=map/entries=] whose [=map/keys=] are "x", "y", "z" and whose [=map/values=] denote uncalibrated magnetic field around the 3 different axes.
+
```

Comment:
any sensor is kind of calibrated in HW... they have to do that because of internal interference in phones etc. What kind of calibration it is, is what matters.